### PR TITLE
BAH-2852 | link ABHA number if ABHA address already exists

### PR DESCRIPTION
**Pull request by kavitha-sundararajan**

### CODE REVIEW

File: src/components/creation/LinkABHAAddress.jsx [hunk 1]
```diff
@@ -77,9 +83,9 @@ const LinkABHAAddress = (props) => {
                         </div>
                             {phrAddressList}
                     </div>
-                    <CheckIdentifierExists id={abhaAddress} setABHAAlreadyExists={setABHAAlreadyExists} />
+                    <CheckIdentifierExists id={abhaAddress} setABHAAlreadyExists={setABHAAlreadyExists} setHealthIdIsVoided={setHealthIdIsVoided} setMatchingPatientUuid={setMatchingPatientUuid} setHealthNumberAlreadyExists={setHealthNumberAlreadyExists}/>
```
Comment (SanoferSameera) [inline src/components/creation/LinkABHAAddress.jsx:86]: setHealthNumberAlreadyExists - is it conveys healthNumber existing in Bahmni or chosen abha address linked patient has health number linked ?

Comment (kavitha-sundararajan) [reply src/components/creation/LinkABHAAddress.jsx:86]: this conveys chosen abha address already has health number linked to it. Shall I change it to `setHealthNumberAlreadyLinked` if this sounds appropriate  or something else?

Comment (SanoferSameera) [reply src/components/creation/LinkABHAAddress.jsx:86]: yeah.

Comment (kavitha-sundararajan) [reply src/components/creation/LinkABHAAddress.jsx]: refactored to `setHealthNumberAlreadyLinked`

**Pull request merged**